args.go

```diff
@@ -343,7 +343,7 @@ func (a *Args) GetUfloatOrZero(key string) float64 {
 // true is returned for "1", "t", "T", "true", "TRUE", "True", "y", "yes", "Y", "YES", "Yes",
 // otherwise false is returned.
 func (a *Args) GetBool(key string) bool {
-	switch string(a.Peek(key)) {
+	switch b2s(a.Peek(key)) {
```

Comment:
Same here, this is also already optimized by the compiler not to allocate: https://go.dev/play/p/fkD2_0estoT

Reply:
done